PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -2715,7 +2717,8 @@ void ScintillaEditView::performGlobalStyles()
 		edgeColor = pStyle->_fgColor;
 	}
 	execute(SCI_SETEDGECOLOUR, edgeColor);
-	::SendMessage(_hParent, NPPM_INTERNAL_EDGEMULTISETSIZE, 0, 0);
+
+	::SendMessage(_hParent, NPPM_INTERNAL_EDGEMULTISETSIZE, 0, (LPARAM)getCurrentBufferID());

```

Comment:
Not sure if you meant to send the buffer ID in NPPM_INTERNAL_EXTERNALLEXERBUFFER but accidentally changed this message instead? I don't think NppBigSwitch uses lParam for this message.